src/sfizz/LFO.cpp

```diff
@@ -70,16 +65,25 @@ void LFO::start(unsigned triggerDelay)
     Impl& impl = *impl_;
     const LFODescription& desc = *impl.desc_;
     const float sampleRate = impl.sampleRate_;
+    const MidiState& state = impl.resources_.midiState;
 
     impl.subPhases_.fill(0.0f);
     impl.sampleHoldMem_.fill(0.0f);
     impl.sampleHoldState_.fill(0);
 
-    const float delay = desc.delay;
+    float delay = desc.delay;
+    for (const auto& mod: desc.delayCC)
+        delay += mod.data * state.getCCValue(mod.cc);
+
     size_t delayFrames = (delay > 0) ? static_cast<size_t>(std::ceil(sampleRate * delay)) : 0u;
     impl.delayFramesLeft_ = triggerDelay + delayFrames;
 
-    impl.fadePosition_ = (desc.fade > 0) ? 0.0f : 1.0f;
+    float fade = desc.fade;
+    for (const auto& mod: desc.fadeCC)
+        fade += mod.data * state.getCCValue(mod.cc);
```

Comment:
Can we get the well-timed value with respect to `triggerDelay` ? same also above